### PR TITLE
eq-205 Multi-choice multi-answer

### PR DESCRIPTION
**Pull request by hamishtaplin**
### Changes
- Adds multiple choice/multiple answer question type (checkboxes)
- radios/checkboxes can optionally be directed to use 2 column layout (via schema)
### How to test
- `npm run compile`
- complete star wars survey, checking that responses are maintained between pages (eg. on submission of errors) and formatted correctly on submission page
### Who can test
- anyone but @hamishtaplin and @weapdiv-david

### CODE REVIEW

**Comment by hamishtaplin** [comment]
@weapdiv-david I'm only getting the first response on the submission page—need to be an array so I can list them out.

File: app/questionnaire/questionnaire_manager.py [hunk 1]
```diff
@@ -70,6 +70,10 @@ def get_rendering_template(self):
     def _process_incoming_post_data(self, post_data):
         user_responses = {}
         user_action = None
+
+        import pdb
```
Comment (dhilton) [inline app/questionnaire/questionnaire_manager.py]: Can we remove the pdb please?

**Comment by dhilton** [comment]
LGTM - Happy to merge, can we squash or rewrite the commits that don't have much description? Makes the commit log a lot easier to work with. 👍

**Comment by hamishtaplin** [comment]
@dhilton Squash away.

**Pull request merged**